test/lsp.py

```diff
@@ -923,8 +938,19 @@ def setup_lsp(self, lsp: JsonRpcProcess, expose_project_root=True):
                 }
             }
         }
+
+        if file_load_strategy != FileLoadStrategy.Undefined:
+            params['initializationOptions'] = {}
+            params['initializationOptions']['file-load-strategy'] = file_load_strategy.value
```

Comment:
```suggestion
            params['initializationOptions'] = { 'file-load-strategy' : file_load_strategy.value }
```

I think that should work

Reply:
Yeah, suggested that myself :) https://github.com/ethereum/solidity/pull/13150#discussion_r951702983